pkgdb/tests/search.bats

```diff
@@ -531,7 +531,7 @@ genParamsNixpkgsFlox() {
 @test "'pkgdb search' doesn't crash when run in parallel" {
   # We don't want other tests polluting parallel test runs so we do this test
   # with a unique cache directory.
-  run --separate-stderr sh -c 'PKGDB_CACHEDIR="$(mktemp -d)" parallel "sleep 0.{}; \"$PKGDB_BIN\" search --ga-registry --match-name hello" ::: $(seq 10)'
+  run --separate-stderr sh -c 'PKGDB_CACHEDIR="$(mktemp -d)" parallel "sleep 1.{}; \"$PKGDB_BIN\" search --ga-registry --match-name hello" ::: $(seq 10)'
```

Comment:
Not sure if this is the intention:
```
seq 10 | parallel echo 1.{}
1.1
1.2
1.3
1.4
1.5
1.6
1.7
1.8
1.9
1.10
```
So if you have only a few CPUs, there is no contention within 0.1s. But if you have >=10 CPUs, you will.

Reply:
Ah, good catch, I'll just do the test with fewer jobs, I don't think running 10 provides us with much more confidence than say 5